sherpa/astro/tests/test_astro.py

```diff
@@ -22,12 +22,20 @@
 import os
 import os.path
 from sherpa.utils import SherpaTestCase, test_data_missing
-from sherpa.utils import has_package_from_list, has_fits_support
-import sherpa.astro.ui as ui
+# from sherpa.utils import has_package_from_list, has_fits_support
+from sherpa.utils import has_fits_support
+from sherpa.astro import ui
 from sherpa.astro.data import DataPHA
 
 logger = logging.getLogger('sherpa')
 
+try:
+    from sherpa.astro import xspec
+    has_xspec = True
+except ImportError:
+    has_xspec = False
+
+# has_xspec = has_package_from_list("sherpa.astro.xspec")
```

Comment:
I can't remember now; I think there was some reason (given I have a commented out `has_package_from_list` line), but I don't know if it was a sensible or correct reason.